src/release/breaking-changes/window-singleton.md

```diff
@@ -0,0 +1,144 @@
+---
+title: The window singleton is deprecated
+description: In preparation for supporting multiple views and multiple windows the window singleton has been deprecated.
+---
+
+## Summary
```

Comment:
This paragraph feels like more suitable for the "Context" section. This section, to my understanding, should summarize the changes and symptoms. For example:

A number of APIs that assume the app only has one view have been removed. Including:

* `GestureBinding.instance.window`.
* <etc>

Accessing these APIs might result in compiling errors such as:

```
Error: Member not found: 'window'.
```

Reply:
Yeah, some of this should move to the context section. I'll do that and provide a more high-level summary of the change itself here.